src/app/checkout/Checkout.tsx

```diff
@@ -100,6 +101,7 @@ class Checkout extends Component<CheckoutProps & WithCheckoutProps & WithLanguag
     stepTracker: StepTracker | undefined;
 
     state: CheckoutState = {
+        billingSameAsShipping: true,
```

Comment:
Is that the default behaviour we want to go to?

Reply:
Yeah, it was the behaviour before the change as well. The purpose of this is to reduce friction in checkout

Reply:
Thanks for the explanation 👍